lib/trento/domain/host/commands/register_host.ex

```diff
@@ -17,5 +17,9 @@ defmodule Trento.Domain.Commands.RegisterHost do
     field :total_memory_mb, :integer
     field :socket_count, :integer
     field :os_version, :string, default: "Unknown"
+
+    field :installation_source, Ecto.Enum,
+      values: [:community, :suse, :unknown],
+      default: :unknown
```

Comment:
is the default needed here in the command? i find it a bit opaque that we can build a command without that field and having a default value in place. this is already managed by the payload parsing, so I would not use any default when building the command.